app/views/layouts/tylium/_sidebar.html.erb

```diff
@@ -2,30 +2,32 @@
 
   <%= render 'layouts/tylium/sidebar_header' %>
 
-  <ul id="main-menu" class="main-menu">
-    <%= content_tag :li, :class => controller_path == 'projects' ? 'active' : '' do %>
-      <%= link_to main_app.project_path(current_project), class: 'sidebar-link', data: { behavior: 'sidebar-link' } do %>
-        <i class="fa fa-tachometer sidebar-link-icon"></i> <span class="sidebar-link-label">Dashboard</span>
+  <div class="mobile-scroll">
```

Comment:
this is the only change in this file, but the indentation changing caused this to look like a lot more changed.

Reply:
The mobile scroll div allows the users to horizontally scroll the nav if there are more icons than the viewport width can fit